### DISCUSSION TITLE
Processing mixed species samples

### DISCUSSION THREAD

**Discussion by prmunn** [GreenleafLab/ArchR#1416] [Questions / Documentation]
I have samples comprised of both human and mouse cells from a mixed species (barnyard) experiment. Is it possible to set up a gene annotation file (using makeTxDbFromEnsembl and createGeneAnnotation) or would I have to analyze them separately (once using hg38 and again using mm10)?

**Comment by rcorces** [comment]
I would not recommend trying to analyze these together. You could do as you propose but this is not an intended use case of ArchR and I imagine it would be an uphill battle.
In case it is helpful - https://github.com/GreenleafLab/ArchR/issues/366

**Reply by prmunn** [reply]
Indeed. I'll analyze them separately.
